pinterestClone/ImagesListFolder/ImagesListService.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ImagesListService.swift
+//  pinterestClone
+//
+//  Created by Денис on 17.05.2023.
+//
+
+import Foundation
+import Kingfisher
+
+class ImagesListService {
+    //MARK: - Public Propeties
+    static let DidChangeNotification = Notification.Name(rawValue: "ImagesListServiceDidChange")
+    
+    let tokenStorage = OAuth2TokenStorage()
+    //MARK: - Private Properties
+    private (set) var photos: [Photo] = []
+    private var lastLoadedPage: Int?
+    private var task: URLSessionTask?
+    private let urlSession = URLSession.shared
+    private let dateFormatter = ISO8601DateFormatter()
+    
+    
+    
+    //MARK: - Ловим Фото из сети
+    func fetchPhotosNextPage() {
+        guard task == nil else {return}
+        let nextPage = lastLoadedPage == nil ? 1 : lastLoadedPage! + 1
+        
+        
+        var urlComponents = URLComponents(string: "https://api.unsplash.com")
+        urlComponents?.path = "/photos"
+        urlComponents?.queryItems = [
+            URLQueryItem(name: "page", value: "\(nextPage)")
+        ]
+        
+        guard let url = urlComponents?.url else { return }
+        
+        var request = URLRequest(url: url)
+        guard let token = tokenStorage.token else {return}
+        request.setValue("Bearer \(token)", forHTTPHeaderField: "Authorization")
+        
+        let dataTask = urlSession.objectTask(for: request) {[weak self] (result: Result<[PhotoResult],Error>) in
```

Comment:
фигурные скобки, перечисления сопровождаются пробелами, чтобы код не слипался 
```suggestion
        let dataTask = urlSession.objectTask(for: request) { [weak self] (result: Result<[PhotoResult], Error>) in
```

Reply:
Ок